coingate/resources/withdrawal.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from decimal import Decimal
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from .platform import Platform
+from .public import Currency
+
+
+class PayoutSetting(BaseModel):
```

Comment:
would it be possible not to strongly type PayoutSetting ? as it can wan have any variations

Reply:
Sure. Updated to `Dict[str, Any]`